HandyNotes Shadowlands Pin Ignore

KnewOne opened the pull request:
Fix for MBB - HandyNotes_Shadowlands bug regarding the Pin from the second addons not being ignored correctly by the MBB.  Further info
MBB/issues/13

KnewOne committed:
HandyNotes Shadowlands Pin Ignore

Fix for MBB - HandyNotes_Shadowlands bug regarding the Pin from the second addons not being ignored correctly by the MBB.  Further info
MBB/issues/13

zarillion commented:
I would *not* go with this solution. This will only ignore the pins from one of my plugins. I also have HandyNotes: Battle for Azeroth, which uses the pin names `HandyNotes_BattleForAzerothPins`. We have unreleased plugins for *every* expansion that we plan on releasing over time. That's why I mentioned the solution must ignore `HandyNotes*Pins` as a pattern to properly ignore them all.

KnewOne committed:
HandyNotes plugins support

A better solution to the previous iteration, focused on long-term support.

KnewOne commented:
Changed the row i've added to support all the plugins. Was hesitant to change the initial line [25] so i went with this approach.
Also improved some ruRU translation.

KnewOne committed:
Ru localization improvements
